lab/experiments/Popover.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+	<div
+		:class="$s.Wrapper"
+	>
+		<m-theme
+			:class="$s.Preview"
+			:theme="theme(backgroundColor)"
+		>
+			<m-heading
+				:size="0"
+			>
+				Parent Site MTheme
+			</m-heading>
+			<input
+				v-model="backgroundColor"
+				type="color"
+			>
+			<m-popover>
+				<template #action="popover">
+					<m-button
+						:class="$s.Button"
+						@click="popover.toggle()"
+					>
+						Popover Toggle
+					</m-button>
+				</template>
+				<template #content>
+					<m-popover-content
+						:class="$s.Popover"
+					>
+						<m-heading>
+							Popover content
+						</m-heading>
+						<m-text>
+							Content for a basic popover
+						</m-text>
+						<m-divider />
+						<m-text>
+							Content for a basic popover
+						</m-text>
+						<m-input
+							variant="outline"
+							placeholder="Delivery address"
+						/>
+					</m-popover-content>
+				</template>
+			</m-popover>
+			<m-text>
+				Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+				sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+				Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+				nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+				reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+				pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa
+				qui officia deserunt mollit anim id est laborum.
+			</m-text>
+			<m-theme
+				:class="$s.Block"
+				:theme="theme(blockColor)"
+			>
+				<m-heading
+					:size="0"
+				>
+					Nested Block MTheme
+				</m-heading>
+				<input
+					v-model="blockColor"
+					type="color"
+				>
+				<m-popover>
```

Comment:
One thought I had was we could make the popover more flexible by passing a profile prop to the component, this is something I thought we could do with sections too -- but for now it makes sense for it to inherit whatever the defaults from it's parent.